8 1 8
[2024-03-30 16:48:16,346] [INFO] [real_accelerator.py:161:get_accelerator] Setting ds_accelerator to cuda (auto detect)
[2024-03-30 16:48:16,347] [INFO] [real_accelerator.py:161:get_accelerator] Setting ds_accelerator to cuda (auto detect)
[2024-03-30 16:48:16,347] [INFO] [real_accelerator.py:161:get_accelerator] Setting ds_accelerator to cuda (auto detect)
[2024-03-30 16:48:16,348] [INFO] [real_accelerator.py:161:get_accelerator] Setting ds_accelerator to cuda (auto detect)
[2024-03-30 16:48:16,350] [INFO] [real_accelerator.py:161:get_accelerator] Setting ds_accelerator to cuda (auto detect)
[2024-03-30 16:48:16,350] [INFO] [real_accelerator.py:161:get_accelerator] Setting ds_accelerator to cuda (auto detect)
[2024-03-30 16:48:16,355] [INFO] [real_accelerator.py:161:get_accelerator] Setting ds_accelerator to cuda (auto detect)
[2024-03-30 16:48:16,360] [INFO] [real_accelerator.py:161:get_accelerator] Setting ds_accelerator to cuda (auto detect)
/hpc2hdd/home/zli755/xueze/reft_ds/Megatron-DeepSpeed/megatron/model/module.py:13: UserWarning: The torch.cuda.*DtypeTensor constructors are no longer recommended. It's best to use methods such as torch.tensor(data, dtype=*, device='cuda') to create tensors. (Triggered internally at ../torch/csrc/tensor/python_tensor.cpp:83.)
  _FLOAT_TYPES = [get_accelerator().FloatTensor(0).dtype]
/hpc2hdd/home/zli755/xueze/reft_ds/Megatron-DeepSpeed/megatron/model/module.py:13: UserWarning: The torch.cuda.*DtypeTensor constructors are no longer recommended. It's best to use methods such as torch.tensor(data, dtype=*, device='cuda') to create tensors. (Triggered internally at ../torch/csrc/tensor/python_tensor.cpp:83.)
  _FLOAT_TYPES = [get_accelerator().FloatTensor(0).dtype]
/hpc2hdd/home/zli755/xueze/reft_ds/Megatron-DeepSpeed/megatron/model/module.py:13: UserWarning: The torch.cuda.*DtypeTensor constructors are no longer recommended. It's best to use methods such as torch.tensor(data, dtype=*, device='cuda') to create tensors. (Triggered internally at ../torch/csrc/tensor/python_tensor.cpp:83.)
  _FLOAT_TYPES = [get_accelerator().FloatTensor(0).dtype]
/hpc2hdd/home/zli755/xueze/reft_ds/Megatron-DeepSpeed/megatron/model/module.py:13: UserWarning: The torch.cuda.*DtypeTensor constructors are no longer recommended. It's best to use methods such as torch.tensor(data, dtype=*, device='cuda') to create tensors. (Triggered internally at ../torch/csrc/tensor/python_tensor.cpp:83.)
  _FLOAT_TYPES = [get_accelerator().FloatTensor(0).dtype]
/hpc2hdd/home/zli755/xueze/reft_ds/Megatron-DeepSpeed/megatron/model/module.py:13: UserWarning: The torch.cuda.*DtypeTensor constructors are no longer recommended. It's best to use methods such as torch.tensor(data, dtype=*, device='cuda') to create tensors. (Triggered internally at ../torch/csrc/tensor/python_tensor.cpp:83.)
  _FLOAT_TYPES = [get_accelerator().FloatTensor(0).dtype]
/hpc2hdd/home/zli755/xueze/reft_ds/Megatron-DeepSpeed/megatron/model/module.py:13: UserWarning: The torch.cuda.*DtypeTensor constructors are no longer recommended. It's best to use methods such as torch.tensor(data, dtype=*, device='cuda') to create tensors. (Triggered internally at ../torch/csrc/tensor/python_tensor.cpp:83.)
  _FLOAT_TYPES = [get_accelerator().FloatTensor(0).dtype]
/hpc2hdd/home/zli755/xueze/reft_ds/Megatron-DeepSpeed/megatron/model/module.py:13: UserWarning: The torch.cuda.*DtypeTensor constructors are no longer recommended. It's best to use methods such as torch.tensor(data, dtype=*, device='cuda') to create tensors. (Triggered internally at ../torch/csrc/tensor/python_tensor.cpp:83.)
  _FLOAT_TYPES = [get_accelerator().FloatTensor(0).dtype]
/hpc2hdd/home/zli755/xueze/reft_ds/Megatron-DeepSpeed/megatron/model/module.py:13: UserWarning: The torch.cuda.*DtypeTensor constructors are no longer recommended. It's best to use methods such as torch.tensor(data, dtype=*, device='cuda') to create tensors. (Triggered internally at ../torch/csrc/tensor/python_tensor.cpp:83.)
  _FLOAT_TYPES = [get_accelerator().FloatTensor(0).dtype]
Traceback (most recent call last):
  File "/hpc2hdd/home/zli755/xueze/reft_ds/Megatron-DeepSpeed/pretrain_zeyu.py", line 366, in <module>
    args.local_rank = local_rank
NameError: name 'args' is not defined
Traceback (most recent call last):
  File "/hpc2hdd/home/zli755/xueze/reft_ds/Megatron-DeepSpeed/pretrain_zeyu.py", line 366, in <module>
    args.local_rank = local_rank
NameError: name 'args' is not defined
Traceback (most recent call last):
  File "/hpc2hdd/home/zli755/xueze/reft_ds/Megatron-DeepSpeed/pretrain_zeyu.py", line 366, in <module>
    args.local_rank = local_rank
NameError: name 'args' is not defined
Traceback (most recent call last):
  File "/hpc2hdd/home/zli755/xueze/reft_ds/Megatron-DeepSpeed/pretrain_zeyu.py", line 366, in <module>
    args.local_rank = local_rank
NameError: name 'args' is not defined
Traceback (most recent call last):
  File "/hpc2hdd/home/zli755/xueze/reft_ds/Megatron-DeepSpeed/pretrain_zeyu.py", line 366, in <module>
    args.local_rank = local_rank
NameError: name 'args' is not defined
Traceback (most recent call last):
  File "/hpc2hdd/home/zli755/xueze/reft_ds/Megatron-DeepSpeed/pretrain_zeyu.py", line 366, in <module>
    args.local_rank = local_rank
NameError: name 'args' is not defined
Traceback (most recent call last):
  File "/hpc2hdd/home/zli755/xueze/reft_ds/Megatron-DeepSpeed/pretrain_zeyu.py", line 366, in <module>
    args.local_rank = local_rank
NameError: name 'args' is not defined
Traceback (most recent call last):
  File "/hpc2hdd/home/zli755/xueze/reft_ds/Megatron-DeepSpeed/pretrain_zeyu.py", line 366, in <module>
    args.local_rank = local_rank
NameError: name 'args' is not defined
srun: error: gpu1-43: tasks 0,3,5: Exited with exit code 1
srun: error: gpu1-43: tasks 1-2,4,6-7: Exited with exit code 1

real	0m4.538s
user	0m0.027s
sys	0m0.025s
